### ISSUE TITLE
Prototype Pollution [High Severity] in lodash

### ISSUE THREAD

**Issue by Misiu** [Vibrant-Colors/node-vibrant#115]
Snyk showed me a couple of errors about the current lodash version.
Patchable issues:

```
  Patch available for lodash@4.17.11
  ✗ Prototype Pollution [Medium Severity][https://snyk.io/vuln/SNYK-JS-LODASH-567746] in lodash@4.17.11
    introduced by node-vibrant@3.1.5 > lodash@4.17.11


Issues with no direct upgrade or patch:
  ✗ Denial of Service (DoS) [Medium Severity][https://snyk.io/vuln/SNYK-JS-JPEGJS-570039] in jpeg-js@0.3.7
    introduced by node-vibrant@3.1.5 > @jimp/types@0.9.3 > @jimp/jpeg@0.9.3 > jpeg-js@0.3.7
  This issue was fixed in versions: 0.4.0
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-LODASH-450202] in lodash@4.17.11
    introduced by node-vibrant@3.1.5 > lodash@4.17.11
  This issue was fixed in versions: 4.17.12
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-LODASH-590103] in lodash@4.17.11
    introduced by node-vibrant@3.1.5 > lodash@4.17.11
  This issue was fixed in versions: 4.17.20
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-LODASH-608086] in lodash@4.17.11
    introduced by node-vibrant@3.1.5 > lodash@4.17.11
  This issue was fixed in versions: 4.17.17
```

The fix is quite simple: `Upgrade lodash to version 4.17.20 or higher.`

**Comment by Misiu** [comment]
@crutchcorn could you take a look at this please?

**Comment by crutchcorn** [comment]
Released on `3.1.6` of `node-vibrant`. Cutting a release for the `develop` versions as well

**Comment by Misiu** [comment]
@crutchcorn thank you 🎉🎉🎉
